flask-api/AuthenticatedHttpClient.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
```

Comment:
In python packages should be named with underscore. -> flask_api
Put CalamariLit in kujira/libs/calamari/